attestation-service/bin/grpc-as/src/server.rs

```diff
@@ -86,17 +92,52 @@ impl AttestationService for Arc<RwLock<AttestationServer>> {
 
         debug!("Evidence: {}", &request.evidence);
 
+        let tee = to_kbs_tee(
+            GrpcTee::from_i32(request.tee)
+                .ok_or_else(|| Status::aborted(format!("Invalid TEE {}", request.tee)))?,
+        );
+        let evidence = STANDARD
+            .decode(request.evidence)
+            .map_err(|e| Status::aborted(format!("Illegal input Evidence: {e}")))?;
+
+        let runtime_data = request
+            .runtime_data
+            .iter()
+            .map(|data_item| {
+                STANDARD
+                    .decode(data_item)
+                    .map_err(|e| Status::aborted(format!("base64 decode runtime data: {e}")))
+            })
+            .collect::<std::result::Result<Vec<Vec<u8>>, Status>>()?;
+
+        let init_data: Vec<Vec<u8>> = request
+            .init_data
+            .iter()
+            .map(|data_item| {
+                STANDARD
+                    .decode(data_item)
+                    .map_err(|e| Status::aborted(format!("base64 decode init data: {e}")))
+            })
+            .collect::<std::result::Result<Vec<Vec<u8>>, Status>>()?;
```

Comment:
Why array of base64 strings? I think these should just be base64 strings. We also need to document the fact that parameters are base64 encoded and document which base64 encoding is used in README.md. In remote APIs it is most commonly `URL_SAFE_NO_PAD`, not `STANDARD`.

Reply:
are there optional parameters in grpc? i care more about how this works for a JSON http API, where we can have can have:
```json
"runtime_data": {}
```
or
```json
"runtime_data": { "data": "" }
```
or skip passing runtime_data.

Reply:
if we stick to an array it would be more like:
* no "runtime_data" or `"runtime_data": []` - we don't check
* `"runtime_data": [""]` we check `hash(b"")`.